commit be347d8e972d46e4b44ad6fcc7a4422d57cbf930
Author: Alejandro Alvarez Ayllon <a.alvarezayllon@gmail.com>
Date:   Wed Dec 18 14:04:49 2019 +0100

    __linux macro is deprecated

diff --git a/ElementsKernel/ElementsKernel/System.h b/ElementsKernel/ElementsKernel/System.h
index 70f2c29..9f7e797 100644
--- a/ElementsKernel/ElementsKernel/System.h
+++ b/ElementsKernel/ElementsKernel/System.h
@@ -99,7 +99,7 @@ const std::string DEFAULT_INSTALL_PREFIX { "/usr" };
 # endif
 #endif
 
-#if defined(__linux) || defined(__APPLE__)
+#if defined(__linux__) || defined(__APPLE__)
 #define TEMPLATE_SPECIALIZATION
 #endif
 
diff --git a/ElementsKernel/src/Lib/System.cpp b/ElementsKernel/src/Lib/System.cpp
index 028b1fe..b6df80d 100644
--- a/ElementsKernel/src/Lib/System.cpp
+++ b/ElementsKernel/src/Lib/System.cpp
@@ -124,7 +124,7 @@ unsigned long unloadDynamicLib(ImageHandle handle) {
 /// Get a specific function defined in the DLL
 unsigned long getProcedureByName(ImageHandle handle, const string& name,
     EntryPoint* pFunction) {
-#if defined(__linux)
+#if defined(__linux__)
   *pFunction = FuncPtrCast<EntryPoint>(::dlsym(handle, name.c_str()));
   if (0 == *pFunction) {
     errno = static_cast<int>(0xAFFEDEAD);
