diff --git a/cmake/doc/Doxyfile.in b/cmake/doc/Doxyfile.in
index 4c5e11a9..fc48301f 100644
--- a/cmake/doc/Doxyfile.in
+++ b/cmake/doc/Doxyfile.in
@@ -1215,7 +1215,7 @@ SERVER_BASED_SEARCH = NO
 # If the GENERATE_LATEX tag is set to YES (the default) Doxygen will
 # generate Latex output.
 
-GENERATE_LATEX = YES
+GENERATE_LATEX = NO
 
 # The LATEX_OUTPUT tag is used to specify where the LaTeX docs will be put.
 # If a relative path is entered the value of OUTPUT_DIRECTORY will be
@@ -1279,7 +1279,7 @@ PDF_HYPERLINKS = YES
 # plain latex in the generated Makefile. Set this option to YES to get a
 # higher quality PDF documentation.
 
-USE_PDFLATEX = YES
+USE_PDFLATEX = NO
 
 # If the LATEX_BATCHMODE tag is set to YES, doxygen will add the \\batchmode.
 # command to the generated LaTeX files. This will instruct LaTeX to keep
