diff --git a/ElementsKernel/CMakeLists.txt b/ElementsKernel/CMakeLists.txt
index 6a88c637..78d40b07 100644
--- a/ElementsKernel/CMakeLists.txt
+++ b/ElementsKernel/CMakeLists.txt
@@ -28,22 +28,22 @@ elements_add_unit_test(Logging tests/src/Logging_test.cpp
 # Path_test
 elements_add_unit_test(PathSearch tests/src/PathSearch_test.cpp
                        EXECUTABLE PathSearch_test
-                       LINK_LIBRARIES ElementsKernel TYPE Boost 
+                       LINK_LIBRARIES ElementsKernel TYPE Boost
                        LABELS Path)
 
 elements_add_unit_test(Path tests/src/Path_test.cpp
                        EXECUTABLE Path_test
-                       LINK_LIBRARIES ElementsKernel TYPE Boost 
+                       LINK_LIBRARIES ElementsKernel TYPE Boost
                        LABELS Path)
 
 elements_add_unit_test(Configuration tests/src/Configuration_test.cpp
                        EXECUTABLE BasicConfiguration_test
-                       LINK_LIBRARIES ElementsKernel TYPE Boost 
+                       LINK_LIBRARIES ElementsKernel TYPE Boost
                        LABELS Path)
 
 elements_add_unit_test(Auxiliary tests/src/Auxiliary_test.cpp
                        EXECUTABLE Auxiliary_test
-                       LINK_LIBRARIES ElementsKernel TYPE Boost 
+                       LINK_LIBRARIES ElementsKernel TYPE Boost
                        LABELS Path)
 
 #-----------------------
@@ -204,19 +204,19 @@ elements_add_test(AddPythonModule COMMAND addPythonModule_test LABELS Creation)
 elements_add_test(AddPythonProgram COMMAND addPythonProgram_test LABELS Creation)
 elements_add_test(AddScript COMMAND addScript_test LABELS Creation)
 
-elements_add_test(CreateDependentElementsProjects 
-                  COMMAND createDependentElementsProjects_test  
+elements_add_test(CreateDependentElementsProjects
+                  COMMAND createDependentElementsProjects_test
                   ENVIRONMENT CMAKE_PREFIX_PATH+=${CMAKE_SOURCE_DIR}/cmake
                   LABELS Creation)
 
 elements_add_test(CheckElementsProjectsTree
-                  COMMAND checkElementsProjectsTree_test  
+                  COMMAND checkElementsProjectsTree_test
                   ENVIRONMENT CMAKE_PREFIX_PATH+=${CMAKE_SOURCE_DIR}/cmake
                   LABELS Creation)
 
 if (NOT SQUEEZED_INSTALL)
   elements_add_test(CheckProjectDependencyFailure
-                    COMMAND checkProjectDependencyFailure_test  
+                    COMMAND checkProjectDependencyFailure_test
                     ENVIRONMENT CMAKE_PREFIX_PATH+=${CMAKE_SOURCE_DIR}/cmake
                     LABELS Creation)
 endif()
@@ -225,23 +225,23 @@ elements_add_test(HEADVersionProject
                   COMMAND HEADVersionProject_test
                   ENVIRONMENT CMAKE_PREFIX_PATH+=${CMAKE_SOURCE_DIR}/cmake
                   LABELS Creation)
-                  
+
 elements_add_test(PatchVersionProject
                   COMMAND patchVersionProject_test
                   ENVIRONMENT CMAKE_PREFIX_PATH+=${CMAKE_SOURCE_DIR}/cmake
                   LABELS Creation)
-                  
-elements_add_test(CMakeOverride 
-                  COMMAND cmakeOverride_test  
+
+elements_add_test(CMakeOverride
+                  COMMAND cmakeOverride_test
                   ENVIRONMENT CMAKE_PREFIX_PATH+=${CMAKE_SOURCE_DIR}/cmake
                   LABELS Creation Override)
-                  
+
 elements_add_test(MissingElementsProjectVersion
                   COMMAND missingElementsProjectVersion_test
                   ENVIRONMENT CMAKE_PREFIX_PATH+=${CMAKE_SOURCE_DIR}/cmake
                   TIMEOUT 300
                   LABELS Creation)
 
-elements_add_test(SqueezedInstall COMMAND squeezedInstall_test
-                  ENVIRONMENT ELEMENTS_SOURCE_DIR=${CMAKE_SOURCE_DIR}
-                  LABELS Creation Meta)
+#elements_add_test(SqueezedInstall COMMAND squeezedInstall_test
+#                  ENVIRONMENT ELEMENTS_SOURCE_DIR=${CMAKE_SOURCE_DIR}
+#                  LABELS Creation Meta)
diff --git a/ElementsServices/tests/python/webdavIsNotInstalled_test.py b/ElementsServices/tests/python/webdavIsNotInstalled_test.py
index 71651716..0d35c137 100644
--- a/ElementsServices/tests/python/webdavIsNotInstalled_test.py
+++ b/ElementsServices/tests/python/webdavIsNotInstalled_test.py
@@ -22,5 +22,5 @@ from ElementsServices.DataSync import WebdavSynchronizer
 class TestWebdavIsNotInstalled(object):
 
     @staticmethod
-    def testWebdavIsNotInstalled(cls):
+    def testWebdavIsNotInstalled():
         assert not WebdavSynchronizer.webdavIsInstalled()
