diff --git a/ElementsServices/tests/python/webdavIsNotInstalled_test.py b/ElementsServices/tests/python/webdavIsNotInstalled_test.py
index 71651716..0d35c137 100644
--- a/ElementsServices/tests/python/webdavIsNotInstalled_test.py
+++ b/ElementsServices/tests/python/webdavIsNotInstalled_test.py
@@ -22,5 +22,5 @@ from ElementsServices.DataSync import WebdavSynchronizer
 class TestWebdavIsNotInstalled(object):
 
     @staticmethod
-    def testWebdavIsNotInstalled(cls):
+    def testWebdavIsNotInstalled():
         assert not WebdavSynchronizer.webdavIsInstalled()
