diff --git a/cmake/ElementsBuildFlags.cmake b/cmake/ElementsBuildFlags.cmake
index 7ac0229..4c7cea3 100644
--- a/cmake/ElementsBuildFlags.cmake
+++ b/cmake/ElementsBuildFlags.cmake
@@ -274,6 +274,8 @@ if(NOT ELEMENTS_DEFAULT_LOGLEVEL)
   endif()
 endif()
 
+option(USE_VERSIONED_LIBRARIES "Generate versioned shared libraries" ON)
+
 #--- Compilation Flags ---------------------------------------------------------
 if(NOT ELEMENTS_FLAGS_SET)
   message(STATUS "Setting cached build flags")
diff --git a/cmake/ElementsProjectConfig.cmake b/cmake/ElementsProjectConfig.cmake
index 946e0d6..be2d477 100644
--- a/cmake/ElementsProjectConfig.cmake
+++ b/cmake/ElementsProjectConfig.cmake
@@ -2440,6 +2440,10 @@ Provide source files and the NO_PUBLIC_HEADERS option for a plugin/module librar
     REQUIRED_LIBRARIES "${ARG_LINK_LIBRARIES}")
   set_property(GLOBAL APPEND PROPERTY LINKER_LIBRARIES ${library})
 
+  if(USE_VERSIONED_LIBRARIES)
+    set_target_properties(${library} PROPERTIES SOVERSION ${CMAKE_PROJECT_VERSION})
+  endif()
+
   #----Installation details-------------------------------------------------------
   install(TARGETS ${library} EXPORT ${CMAKE_PROJECT_NAME}Exports DESTINATION ${CMAKE_LIB_INSTALL_SUFFIX} OPTIONAL)
   elements_export(LIBRARY ${library})
