diff --git a/cmake/ElementsLocations.cmake b/cmake/ElementsLocations.cmake
index a2591ce..bf463eb 100644
--- a/cmake/ElementsLocations.cmake
+++ b/cmake/ElementsLocations.cmake
@@ -4,7 +4,7 @@ include(SGSPlatform)
 include(ElementsBuildFlags)
 
 if(HIDE_SYSINC_WARNINGS)
-  set(CMAKE_NO_SYSTEM_FROM_IMPORTED FALSE)  
+  set(CMAKE_NO_SYSTEM_FROM_IMPORTED FALSE)
 else()
   set(CMAKE_NO_SYSTEM_FROM_IMPORTED TRUE)
 endif()
@@ -13,7 +13,7 @@ if(NOT DEFINED SQUEEZED_INSTALL)
     set(SQUEEZED_INSTALL ON
         CACHE STRING "Enable the squizzing of the installation into a prefix directory"
         FORCE)
-    message(STATUS "Sets the default value for SQUEEZED_INSTALL to ${SQUEEZED_INSTALL}")     
+    message(STATUS "Sets the default value for SQUEEZED_INSTALL to ${SQUEEZED_INSTALL}")
 endif()
 
 # Install Area business
@@ -45,7 +45,7 @@ if(NOT SQUEEZED_INSTALL)
     set(ELEMENTS_BASE_DIR "/opt/euclid" CACHE STRING "Euclid Base Install Directory")
     message(STATUS "${ELEMENTS_BASE_VAR} is not in the environment: using default ${ELEMENTS_BASE_DIR}")
   endif()
-  
+
 endif()
 
 
@@ -83,8 +83,8 @@ if(SQUEEZED_INSTALL)
   endif()
 endif()
 
-set(CMAKE_LIB_INSTALL_SUFFIX ${lib_install_suff} CACHE STRING "Suffix for the install directory of the libraries" FORCE)
-set(CMAKE_BIN_INSTALL_SUFFIX bin CACHE STRING "Suffix for the install directory of the binaries" FORCE)
+set(CMAKE_LIB_INSTALL_SUFFIX ${lib_install_suff} CACHE STRING "Suffix for the install directory of the libraries")
+set(CMAKE_BIN_INSTALL_SUFFIX bin CACHE STRING "Suffix for the install directory of the binaries")
 
 
 
@@ -184,7 +184,7 @@ get_arch_lib_dir(that_arch)
 file(TO_CMAKE_PATH "$ENV{CMAKE_PREFIX_PATH}" current_cmake_prefix_path)
 
 set(ELEMENTS_DEFAULT_SEARCH_PATH)
-foreach(_ds ${current_cmake_prefix_path})  
+foreach(_ds ${current_cmake_prefix_path})
   list(APPEND ELEMENTS_DEFAULT_SEARCH_PATH ${_ds}/${that_arch}/cmake/ElementsProject)
 endforeach()
 set(ELEMENTS_USR_SEARCH_PATH /usr/${that_arch}/cmake/ElementsProject)
